### DISCUSSION TITLE
Vote to retire @kvedurmu

### DISCUSSION THREAD

**Discussion by kvedurmu** [paketo-buildpacks/feedback#94] [Content Team]
Hey @paketo-buildpacks/content-maintainers, I'd like to propose to move @kvedurmu  to an emeritus maintainer position. Kashyap has made some great contributions, but is now less involved in the project. We'd like to make sure his contributions are recognized. He will receive an emeritus status recognition on https://github.com/paketo-buildpacks/community. He will then also be removed from his current position as an active maintainer to the Content team.

Can I get a 👍 / 👎 vote?

cc/ @ryanmoran

**Comment by ryanmoran** [comment]
With a unanimous vote to approve, @kvedurmu will be retired from the Content Maintainers team.
